### ISSUE TITLE
Hi,GUYS，I create a new Electrical rewind

### ISSUE THREAD

**Issue by muzixiaoyang** [moggieuk/Happy-Hare#147]
[https://github.com/muzixiaoyang/3Dprinter-Electric-rewind---filament-rewind---multi-color-printing](url)
[https://www.bilibili.com/video/BV1MT4y1n7Rv/?spm_id_from=333.999.0.0](url)

**Comment by moggieuk** [comment]
Wow. That is very nice!

I watched the video and it looks very clever.  Have you shared on the Voron Discord?

**Comment by muzixiaoyang** [comment]
Thank you sir, I haven't shared VORON Discord yet. I am preparing to translate it and share it on Discord. At present, I have prepared your Type-B hardware

**Comment by moggieuk** [comment]
Great. HH support for "Type-B" solutions is in alpha.  The main code I need time to test is the drive stepper switching logic..

**Comment by muzixiaoyang** [comment]
Thank you, sir, I have prepared two TYPE-B schemes and created two new PCB boards, ams and ams lite, maybe we can communicate further :D

![AMS-LITE](https://github.com/moggieuk/Happy-Hare/assets/116904500/d188104a-9126-4c86-a445-68cc7b01a924)
![TU](https://github.com/moggieuk/Happy-Hare/assets/116904500/b4783471-b168-41db-9545-d37f0cb61330)ding FU.JPG…]()
![AMS](https://github.com/moggieuk/Happy-Hare/assets/116904500/a2dc7ac3-a0be-46ae-bc87-1a1eb7843945)

**Comment by thiagolocatelli** [comment]
WOW! This looks nice! Reversed engineered the AMS feeder. Clever.

**Comment by moggieuk** [comment]
> Thank you, sir, I have prepared two TYPE-B schemes and created two new PCB boards, ams and ams lite, maybe we can communicate further :D

Happy to keep in touch because I can see this being a popular edition and I'd love to be able to drive it with Happy Hare...

**Comment by muzixiaoyang** [comment]
Yes sir, I am sorting out the repository of AMS-LITE

**Comment by thiagolocatelli** [comment]
@muzixiaoyang do you have new videos?

**Comment by muzixiaoyang** [comment]
Yes, I have the latest introductory video, but in Chinese, currently preparing to shoot a video in English, sir
https://www.bilibili.com/video/BV1Mv421i7iz

**Comment by thiagolocatelli** [comment]
@muzixiaoyang that is not very far from what I had working! You have made way more progress. Also, Mellow has a new board that you can use on your project too. I have been beta testing it.


[Video Loading filament](https://cdn.discordapp.com/attachments/1155906995950465146/1155922703040122960/IMG_8214.mov?ex=65d39368&is=65c11e68&hm=b3dd59016265f412f49ffea3a5e9037feaf66fde1a57b9b618b515a701d21705&)


[Video Homing filament to toolhead](https://cdn.discordapp.com/attachments/1155906995950465146/1160952392766988391/IMG_8412.mov?ex=65d36aab&is=65c0f5ab&hm=85e357947d675d02429ffb5cfcb1e78f6a430d3c4f040e04205979982fbf3ec1&)


[Video Buffer/4-in-1-out internals](https://cdn.discordapp.com/attachments/1155906995950465146/1160021782166765619/IMG_8389.mov?ex=65d007f9&is=65bd92f9&hm=bc3970a1b5a8114891fe5ee84c0a632f7c136f36773b4c657f0111be55af27da&)


<img width="760" alt="Screenshot_2023-10-10_at_1 46 12_PM" src="https://github.com/moggieuk/Happy-Hare/assets/1017031/c1025864-8acd-4f23-97cb-6eb8edd5415d">

<img width="980" alt="Screenshot_2023-10-04_at_9 53 36_PM" src="https://github.com/moggieuk/Happy-Hare/assets/1017031/b8523204-0958-4a55-a58d-d78e464e777e">

<img width="1002" alt="Screenshot_2023-10-03_at_9 35 58_PM" src="https://github.com/moggieuk/Happy-Hare/assets/1017031/4483d270-6efe-4a37-9a7f-ab94073e755f">

**Comment by muzixiaoyang** [comment]
Great,sir,i make a new 4-in-1-out,maybe I can share it with you :D
![4in1out-hub](https://github.com/moggieuk/Happy-Hare/assets/116904500/0f3de9f1-4377-43b4-b0ae-efb84dfafb6e)

**Comment by muzixiaoyang** [comment]
Hi sir, can I add your discord?

**Comment by thiagolocatelli** [comment]
My discord is ".doublet" or just search for DoubleT

**Comment by muzixiaoyang** [comment]
Okay, I added
My discord is "muzixiaoyang"
Looking forward to communicating with you
